Automattic/wp-calypso
# Open Sites view in Subscription Manager for internal EN users

Fixes https://github.com/Automattic/wp-calypso/issues/76280

## Proposed Changes

This PR opens the Sites view in the Subscription Manager to all internal users (users with a WordPress account) in English.

## Testing Instructions

1. Apply this PR and start the application.
2. With the WordPress user logged in in your browser, go to `http://calypso.localhost:3000/subscriptions/sites`. You should see the Sites view in Calypso.
3. Then check that clicking on `Comments` and `Pending` tabs takes you to `https://wordpress.com/email-subscriptions/?option=comments&locale=en` and `https://wordpress.com/email-subscriptions/?locale=en&option=pending` respectively.
4. Go directly to `http://calypso.localhost:3000/subscriptions/comments` and `http://calypso.localhost:3000/subscriptions/pending` and check that it also takes you to https://wordpress.com/email-subscriptions/?option=comments&locale=en` and `https://wordpress.com/email-subscriptions/?locale=en&option=pending` respectively.
5. Then, check all those links and clicks with an external EN user.
